README.template.md

```diff
@@ -355,7 +358,11 @@ mkcert \
 
 ## Upgrading Isle Docker Images
```

Comment:
This is ambiguous and could mean:
* upgrading from ISLE to site template
* packaging updates that you've made to ISLE docker images
It also seems implied that you could be using non-ISLE docker images, and that seems not to be spelled out here.